charts/internal/shoot-system-components/charts/rook-ceph/templates/crds.yaml

```diff
@@ -0,0 +1,1369 @@
+###################################################################################################################
+# Create the CRDs that are necessary before creating your Rook cluster.
+# These resources *must* be created before the cluster.yaml or their variants.
+###################################################################################################################
+# OLM: BEGIN CEPH CRD
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
```

Comment:
You could move the CRDs into a separate chart like it's done for other provider extensions, see https://github.com/gardener/gardener-extension-provider-aws/tree/master/charts/internal/shoot-crds and https://github.com/gardener/gardener-extension-provider-aws/pull/337

Reply:
I am a little concerned that these are not general CRDs, but tied closely to rook-ceph. I would prefer to leave it, if it works.

Reply:
I'm not too much into the details, but I think leaving them here can potentially lead to issues in the cluster deletion case. https://github.com/gardener/gardener/pull/3562 and the references have more information about it.

Reply:
I skimmed it. It sounds like it is a deletion issue. I am not very comfortable pulling elements that are important just for rook-ceph (and critical to it) out of its own chart directory. I am content leaving it as a future fix, if it comes up?